zcash_client_backend/src/wallet.rs

```diff
@@ -30,15 +30,44 @@ pub struct WalletTx<N> {
     pub shielded_outputs: Vec<WalletShieldedOutput<N>>,
 }
 
+#[derive(Debug, Clone)]
 pub struct WalletTransparentOutput {
-    pub outpoint: OutPoint,
-    pub txout: TxOut,
-    pub height: BlockHeight,
+    outpoint: OutPoint,
+    txout: TxOut,
+    height: BlockHeight,
+    recipient_address: TransparentAddress,
 }
 
 impl WalletTransparentOutput {
-    pub fn address(&self) -> TransparentAddress {
-        self.txout.script_pubkey.address().unwrap()
+    pub fn from_parts(
```

Comment:
Should really be documented, but in the interest of not blocking the SDKs I won't block on it this time.